src/storage/store/engine_tests.rs

```diff
@@ -1236,6 +1219,8 @@ mod tests {
         commit_message(&mut engine, &username_add).await;
     }
 
+    // this test needs to be updated to use an actual transfer event since validation logic checks the fname signer signature
+    #[ignore]
```

Comment:
Hmm, we should actually pull the signer address from the validations from a config, so tests can pass in a custom public key.